utilities/llmd_constants.py

```diff
@@ -35,6 +35,7 @@ class ModelStorage:
     TINYLLAMA_S3: str = SharedModelStorage.S3.TINYLLAMA
     S3_QWEN: str = SharedModelStorage.S3.QWEN_7B_INSTRUCT
     HF_TINYLLAMA: str = SharedModelStorage.HuggingFace.TINYLLAMA
+    HF_OPT125M: str = SharedModelStorage.HuggingFace.OPT125M
```

Comment:
hey @VedantMahabaleshwarkar  try using the TINYLLAMA model, please

Reply:
While testing yesterday I found that the TinyLlama model was sometimes timing out while trying to perform inference. Not sure if that was due to the cluster I had, but the opt125m model was more consistent